### DISCUSSION TITLE
App URL has format *https://test.codemagic.app/#/

### DISCUSSION THREAD

**Discussion by AyoParadis** [codemagic-ci-cd/codemagic-docs#1097] [Q&A] [answered]
I'm running a flutter app in the browser.

Can we remove the ``#/`` at the end of the URL for all routes? I can't seem to figure out how.

**Comment by himesh-cm** [comment, marked as answer]
Hi @AyoCodess, This issue is related to flutter and not related to Codemagic.

I found this SO [Link](https://stackoverflow.com/questions/59870357/how-to-remove-hash-from-url-in-flutter-web).

Please check if it's helpful.

**Reply by himesh-cm** [reply]
Hi @AyoCodess, can you please confirm if you still face this issue?

**Reply by AyoParadis** [reply]
No i am not, we can close this now. Thank you
